src/lakefs_spec/spec.py

```diff
@@ -386,7 +386,7 @@ def info(self, path: str, **kwargs: Any) -> dict[str, Any]:
         self.run_hook(FSEvent.INFO, HookContext.new(path))
         return statobj
 
-    def ls(self, path, detail=True, **kwargs):
+    def ls(self, path: str, detail: bool = True, **kwargs: Any) -> list[Any]:
```

Comment:
I'd be fine with `list`, since it is equivalent to `list[Any]`

Reply:
Changed in [cc14128](https://github.com/appliedAI-Initiative/lakefs-spec/pull/123/commits/cc141285497fdb326d1aa4e3aae20edbe28ce25e).

Reply:
No, it's not ;)

Reply:
Sorry, meant [cc14128](https://github.com/appliedAI-Initiative/lakefs-spec/pull/123/commits/cc141285497fdb326d1aa4e3aae20edbe28ce25e).